test/sanity.js

```diff
@@ -81,3 +82,27 @@ transformFile('test/fixtures/test-no-svg-or-react.js', {
     throw new Error('Test failed: React import was present');
   }
 });
+
+
+function transform(testName) {
+  return callback =>
+    transformFile(`test/fixtures/test-${testName}.jsx`, {
+      presets: ['airbnb'],
+      plugins: [
+        '../../src/index',
+      ],
+    }, callback);
+}
+
+async.parallel(
```

Comment:
I removed the async dep in favor of promises. I conserved the parallel nature of it because it saves on tabs :stuck_out_tongue:. Let me know what you think.

Reply:
Sorry, that doesn't work because a throw from a promise is just a rejected promise instead of an actual throw, I'll remove the promises all together.

Reply:
i guess i'm a little confused why we need the callback at all; wont it be sync if the callback isn't provided?

Reply:
I wasn't aware of the sync way of using `transformFile`. Why do you even use it async on all other tests?

Reply:
oh, you're right - it's because the order of the other tests resolving doesn't matter. Does it matter for these two?

Reply:
not really.